intTests/test_type_errors/err024.log.good

```diff
@@ -0,0 +1,6 @@
+ Loading file "err024.saw"
+ err024.saw:3:9-3:10: Unbound variable: "y" (/Users/dholland/projects/mtv/shub/saw-script/intTests/test_type_errors/err024.saw:3:9-3:10)
```

Comment:
Surely we don't want `/Users/dholland/...` paths to leak into the `.good` files that we check in? (Similar comments apply to other `.good` files as well.)